packages/ec-core/src/operator/mod.rs

```diff
@@ -1,3 +1,10 @@
+//! Documentation on the `operator` module...
```

Comment:
Did we mean to write something here or is this intended as a placeholder?

Reply:
This is really just a placeholder and does need to be filled in. The `Operator` module is actually pretty complex (and can lead to some tricky compile errors) and so does need to be documented with some care, but I didn't want to tie up these "simple" PRs with that requirement.